charts/app-config/values-integration.yaml

```diff
@@ -1691,6 +1716,7 @@ govukApplications:
     ingress:
       enabled: true
       annotations:
+        <<: *default-alb-ingress-annotations
         alb.ingress.kubernetes.io/security-groups: eks_ingress_www_origin
         alb.ingress.kubernetes.io/load-balancer-name: www-origin
         alb.ingress.kubernetes.io/healthcheck-path: /readyz
```

Comment:
Can we remove this as it's now in `default-alb-ingress-annotations`?

Reply:
Oh yeah well spotted, thanks! Done!